### PR TITLE
pull request for contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by Miliyonayalew**
Added a contact form to the project
Added an action and method attribute to the form
Added ul, li, input field to the form
Added HTML5 validations as per the requirement
Collected data by using the Formspree service
Added styles to contact form using flexbox
Implemented button interaction

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -295,26 +295,30 @@ <h6 class="title-tag-text">Terminal</h6>
...
+                    </li>
+                    <li class="email">
+                            <input class="email-address" id="email" type="email" placeholder="Email address" required>
+                    </li>
+                    <li class="feedback">
+                        <textarea class="textarea"  name="feedback" placeholder="Enter text here" id="feedback" cols="30" rows="10" maxlength="500" required></textarea>
+                    </li>
+                </ul>
                <button type="submit" class="get-in-touch">Get in touch</button>
             </form>
```
Comment (Kamba56) [inline index.html]: - [ ] Please you should stick to the design as much as possible. Kindly make your colors match the design and also add the background image
*Your design*
Screenshot 2022-08-23 223110

*Figma template*
Screenshot 2022-08-23 223237


- [ ] Please make sure the hover and pressed state on your buttons matches that of the design template.
Screenshot 2022-08-20 203951
-

File: index.html [hunk 2]
```diff
@@ -295,26 +295,30 @@ <h6 class="title-tag-text">Terminal</h6>
...
-            <div class="social-icons bottom">
+            <div class="social-icons-bottom">
                 <a href="https://twitter.com/MilaAyalew"><i class="fa-brands fa-twitter"></i></a>
                 <a href="https://www.linkedin.com/in/miliyon-ayalew-210808131/"><i class="fa-brands fa-linkedin-in"></i></a>
                 <a href="https://github.com/Miliyonayalew/"><i class="fa-brands fa-github"></i></a>
                 <a href=""><i class="fa-brands fa-angellist"></i></a>                
             </div>
```
Comment (Kamba56) [inline index.html]: - [Optional] Consider using an unordered list to list out your social icons. This is best practice for HTML semantics.😁

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @Miliyonayalew, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
AlmostThereGIF

## Highlights
- [x] All linters checks passed
- [x] Descriptive PR title and description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [Optional] It's better to follow commit message standards by making them capitalized and in the imperative form (Example: 'Modify readme file' instead of 'modified readme'). This way, your commit history will be more readable and more professional ✨

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
### Hi @Miliyonayalew , 
### I'm Levy, your PR reviewer

Excellent for implementing the requested changes. :sunglasses: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 💯🌟 

5SW

## Status: APPROVED 🔥 🔥 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**